prologin/homepage/templates/homepage/stub-semifinal-result.html

```diff
@@ -55,6 +55,14 @@
     {% endif %}
   {% endif %}
 {% else %}
+  {# ER2021 PLEASE DELETE AFTER #}
+  {% if wish_override_allowed %}
+      <div class="alert alert-warning">
+        <p>{% blocktrans with date_end=wish_override_allowed_until %}You can select your semifinal event until {{ date_end }}{% endblocktrans %}</p>
+        <p><a href="{% url 'contest:semifinal-event-selection' %}" class="btn btn-primary btn-block">{% trans "Choose my semifinal event" %}</a></p>
+        <p><strong>{% trans "Currently selected event" %} : {{ wish_override_first_wish }}</strong></p>
```

Comment:
Use a blocktrans :-)
```suggestion
        <p><strong>{% blocktrans with width=wish_override_first_wish %}You've selected {{ wish }}.{% endblocktrans %}</strong></p>
```